lib/govuk_navigation_helpers/tasklist/helper.rb

```diff
@@ -0,0 +1,55 @@
+module GovukNavigationHelpers
+  module Tasklist
+    module Helper
+      def self.included(base)
+        base.helper_method(
+          :show_tasklist_header?,
+          :show_tasklist_sidebar?,
+        )
+      end
+
+      def current_tasklist_content(tasklist_content)
+        @current_tasklist_content = tasklist_content
+      end
+
+      def show_tasklist_header?
+        if defined?(should_show_tasklist_header?)
+          should_show_tasklist_header?
+        end
+      end
+
+      def show_tasklist_sidebar?
+        if defined?(should_show_tasklist_sidebar?)
+          should_show_tasklist_sidebar?
+        end
+      end
+
+      def configure_current_task
+        set_task_as_active_if_current_page
+      end
+
+      def set_task_as_active_if_current_page
```

Comment:
I'd love a test around this, because we really need to know if this is going to break because we've changed the file format.